tensorflow/g3doc/resources/roadmap.md

```diff
@@ -26,11 +26,10 @@ we do not have timelines for these features.
 * Repeated partial graph evaluation ([#672](https://github.com/tensorflow/tensorflow/issues/672))
 
 ### Platforms
-* iOS support ([#16](https://github.com/tensorflow/tensorflow/issues/16))
 * OpenCL support ([#22](https://github.com/tensorflow/tensorflow/issues/22))
 * Windows support ([#17](https://github.com/tensorflow/tensorflow/issues/17))
-* MacOS GPU support
 
 ### Community
-* Integration with other machine learning frameworks
-* Better installation support; support for package managers
+* More educational resources
+* Better integration of TensorFlow into the opensource big data ecosystem ([#2655](https://github.com/tensorflow/tensorflow/issues/2655), [#2218](https://github.com/tensorflow/tensorflow/issues/2218))
```

Comment:
Ah, thank. Yes, definitely #1996. Probably not #2126 here though, but it should be added to core. I'll make a PR.